tests/TestOne/PackageA/package.json

```diff
@@ -4,7 +4,11 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "PackageB": "../PackageB"
+    "PackageB": "../PackageB",
+    "buildtool": "../buildtool"
```

Comment:
A simplest pjc-compat build tool is also modelled as being distributed as npm package.